src/microbe_stage/IEngulfable.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+
+/// <summary>
+///   Objects that can be engulfed by a microbe.
+/// </summary>
+[UseThriveSerializer]
+public interface IEngulfable : IGraphicalEntity
+{
+    /// <summary>
+    ///   The size of this engulfable object based on microbe hex count.
+    /// </summary>
+    public float Size { get; }
+
+    public float Radius { get; }
+
+    public EntityReference<Microbe> HostileEngulfer { get; }
+
+    /// <summary>
+    ///   The current step of phagocytosis process this engulfable is currently in.
+    /// </summary>
+    public PhagocytosisPhase PhagocytizedStep { get; set; }
```

Comment:
```suggestion
    public PhagocytosisPhase PhagocytosisStep { get; set; }
```
Would that be a better name?